contracts/package.json

```diff
@@ -58,6 +59,7 @@
     "hardhat-deploy": "^0.11.30",
     "hardhat-deploy-ethers": "^0.3.0-beta.13",
     "hardhat-gas-reporter": "^1.0.9",
+    "hardhat-network-helpers": "^0.0.1",
```

Comment:
Isn't this a duplicate of `@nomicfoundation/hardhat-network-helpers`?

Reply:
Yes, thanks removing